### DISCUSSION TITLE
2007/05/11/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-6-%E7%83%AD%E5%B8%A6%E5%B2%9B%E5%B1%BF%E7%9A%84%E6%A4%8D%E7%89%A9

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#158] [Announcements]
# 2007/05/11/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-6-%E7%83%AD%E5%B8%A6%E5%B2%9B%E5%B1%BF%E7%9A%84%E6%A4%8D%E7%89%A9

前一段时间睡眠质量还不错，有阵子没做离奇古怪的梦了。这直接迟滞了一个伟大科幻作家的诞生:)

https://blog.qizhen.xyz/2007/05/11/%E8%AE%B0%E5%BD%95%E6%A2%A6%E5%A2%83-6-%E7%83%AD%E5%B8%A6%E5%B2%9B%E5%B1%BF%E7%9A%84%E6%A4%8D%E7%89%A9

**Comment by ruanqizhen** [comment]
之前博客下的留言：


wen
2007年05月15日 13:20 编辑
看到你的留言后我很开心，虽然没有我想象的那么有意思，不过也算满足了我的好奇心
谢谢了

回复
杰
2007年05月14日 19:41 编辑
你下次写blog的时候，可以先用文本编辑器，例如WORD,UNTRAEDIT这样的工具进行。
然后再进入编辑模式，粘贴上去就可以了。
免得让我们看见：“此空间暂时不可用！”的提示。  ：）

回复
秋
2007年05月14日 10:32 编辑
梦是反应一个人的真实境况~
书上说是右脑的作用~~

回复
wen
2007年05月13日 20:21 编辑
你的梦很有意思，很像“小王子”里的情景
我也经常做奇怪的梦，不会是因为这个你才加我的吧嘿嘿

回复
wen
2007年05月13日 20:16 编辑
很不明白你为什么要加我为好友呢？？我觉的我们生活在不同的世界里啊
能告诉我原因吗？我很想知道。可在我空间留言
 

回复
思源
2007年05月12日 17:33 编辑
你是谁阿？
 
好奇怪，我在你的朋友里，也有人从这里访问我的空间。
 
联系联系，我是马，你呢？

回复
楠
2007年05月12日 03:05 编辑
以后不到你的cube附近去了，太危险了

回复
Wu
2007年05月11日 11:32 编辑
太有才了！

回复
arrowpig
2007年05月11日 08:46 编辑
你想说什么？
